constants/config.go

```diff
@@ -0,0 +1,14 @@
+package constants
+
+import mongik "github.com/FrosTiK-SD/mongik/models"
+
```

Comment:
Update the `Readme.md` of the project

Make some documentation as its a breaking change